packages/svelte-docgen/src/parser/type/constructible.test.ts

```diff
@@ -340,11 +340,11 @@ describe("Constructible", () => {
 			            "alias": "Iterable",
 			            "kind": "interface",
 			            "members": Map {
-			              "__@iterator@19" => {
```

Comment:
Note to self for the future. I can't figure out what exactly is causing those numbers to change.

Not to mention, I'd rather have a better way to display this type _(which should be `Iterator`)_